roles/idm-host-cert/README.md

```diff
@@ -0,0 +1,100 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with creating and configuring idm host certs.
+
+First it logs into idm, creates a session, and then registers the host. Then, it generates a private key and csr. Next, it creates the cert, retrieves the old one, and overwrites it. As a final check, it prints out the certs.
```

Comment:
I'd update this to read:
`A certificate is created by adding the FQDN for the requested certificate as a host, then the CSR is created and submitted before the host certificate & key (+ CA certificate) are retrieved. The certificates can optionally be written to local files for consumption.`